diyepw/create_amy_epw_files_for_years_and_wmos.py

```diff
@@ -0,0 +1,83 @@
+import pandas as pd
+import os
+from typing import List, Tuple
+from .create_amy_epw_file import create_amy_epw_file
+from ._logging import _logger
+
+def create_amy_epw_files_for_years_and_wmos(
+        wmo_indices: List[int],
+        years: List[int],
+
+        *,
+        max_records_to_interpolate: int,
+        max_records_to_impute: int,
+        max_missing_amy_rows: int,
+        amy_epw_dir: str = None,
+        tmy_epw_dir: str = None,
+        amy_dir: str = None,
+        amy_files: Tuple[str, str] = None,
+        allow_downloads: bool = False
```

Comment:
defaulting `allow_downloads` to True might be reasonable, unless there's reason for not

Reply:
Yeah, I didn't actually want allow_downloads at all, because the ability to automatically download files from NOAA and EnergyPlus and use them to construct EPWs is the whole point of the library. But Chris Vernon objected to having a library that downloaded data without being given explicit permission to do so, and allow_downloads with a False default was the resulting feature. I'd also prefer that it at least be True by default, but that's a team policy decision rather than an engineering one.